modules/mobile/spec/request/threads_request_spec.rb

```diff
@@ -91,5 +91,16 @@
 
       expect(response).to have_http_status(:bad_request)
     end
+
+    it 'responds to PATCH threads/move' do
+      VCR.use_cassette('sm_client/threads/moves_a_thread_with_id') do
+        patch '/mobile/v0/messaging/health/threads/7065799/move?folder_id=0',
+              headers: iam_headers
+      end
+
+      puts response
+      expect(response).to be_successful
+      expect(response).to have_http_status(:no_content)
+    end
```

Comment:
Maybe others can weigh in, but there may need to be a failure case test, such as what happens when you attempt to move to a non-existent folder

Reply:
Yes. The upstream team did not capture this information in VCRs, specs, or docs. We should either contact them and request a VCR for the not found case or use their controller on staging to figure out what it would return and create our own VCR.